src/tasks/WasmAppBuilder/WasmAppBuilder.cs

```diff
@@ -168,26 +71,40 @@ protected override bool ExecuteInternal()
         }
         MainAssemblyName = Path.GetFileName(MainAssemblyName);
 
-        var config = new WasmAppConfig()
+        var config = new BootJsonData()
         {
-            MainAssemblyName = MainAssemblyName,
-            GlobalizationMode = InvariantGlobalization ? "invariant" : HybridGlobalization ? "hybrid" : "icu"
+            config = new(),
+            entryAssembly = MainAssemblyName,
+            icuDataMode = InvariantGlobalization
```

Comment:
I would suggest moving this into a method, with comments on what we are trying to do here. That will also help to check whether we have tests for all the cases or not.

Reply:
I have rewritten it and added comments. Please take a look whether I should further improve it.

Reply:
Looks good. 👍